lib/utils/__tests__/isStandardSyntaxDeclaration.test.js

```diff
@@ -166,6 +166,16 @@ describe('isStandardSyntaxDeclaration', () => {
 			true,
 		);
 	});
+
+	it('supports root-level declarations', () => {
+		const doc = new postcss.Document();
+		const root = postcss.parse('color: yellow;');
+
+		root.parent = doc;
+		doc.nodes.push(root);
+
+		expect(isStandardSyntaxDeclaration(root.nodes[0])).toBe(true);
```

Comment:
[suggest] If we'd like to focus on "root-level", using `Document` seems confusing. Instead, what about simplifying the test code like this?

```suggestion
		expect(isStandardSyntaxDeclaration(decl('color: yellow;'))).toBe(true);
```

Reply:
no preference here but fyi the reason i did it this way was basically to ensure there's a test somewhere that makes sure it works with declarations deeply contained in a document (don't think there are any such tests yet).

at the min the implementation doesn't care, it was just in case someone some day updates this function and breaks it for those cases. its not actually testing any existing code branch

Reply:
I've applied the suggestion via 54a5098. Thanks.